src/service/QueryInstance.cpp

```diff
@@ -133,22 +133,7 @@ void QueryInstance::onError(Status status) {
         case Status::Code::kPermissionError:
             rctx->resp().set_error_code(cpp2::ErrorCode::E_BAD_PERMISSION);
             break;
-        case Status::Code::kBalanced:
```

Comment:
Why change this ?

Reply:
We need not to update here each time when storage add new error status. Only handling our concerned error code is enough.